File: 9.py
import math

for a in xrange(2, 997):
	for b in xrange(a, 998):
		c = math.sqrt((a**2)+(b**2))
		if ((a + b) + c) == 1000:
			print a
			print b
			print c
			print a * b * c
			

File: 9.js
var a,b,c;

for (a = 2; a <= 997; a++) {
    for (b = a; b <= 998; b++) {
        c = Math.sqrt( (a * a) + (b * b) )
        if (a+b+c == 1000) {
            console.log(a,b,c,a*b*c);
        }
    }
}